### PR TITLE
Add documenation about confirmation email and fix broken badge

### PR DESCRIPTION
**Pull request by sonalkr132**
Changing codeclimate badge to what is said [here](https://codeclimate.com/github/rubygems/rubygems.org/badges#maintainability-markdown). One provided on https://shields.io/ for codeclimate is not working.
Code Climate

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -115,6 +115,13 @@ Follow the instructions below on how to install Bundler and setup the database.
 * Start memcached: `memcached`
 * Run the tests: `bundle exec rake`
 
+#### Confirmation emails
+
+* Start rails: `rails s`
+* Sign up for a new user: http://localhost:3000/sign_up
+* Account confirmation email: http://localhost:3000/rails/mailers/mailer/email_confirmation
```
Comment (bronzdoc) [inline CONTRIBUTING.md]: I think we need to specify a that the url needs the confirmatrion token.
`ttp://localhost:3000/rails/mailers/mailer/email_confirmation/<confirmation-token>`

**Comment by bronzdoc** [comment]
@sonalkr132 do you think it would be a good idea to create a script under the `script` folders that that generates a user in the db?
(i could work on it )

File: CONTRIBUTING.md (hunk 1 above)
Comment (sonalkr132) [reply CONTRIBUTING.md]: No. It is url where you can see the preview actual confirmation email. User can just click the confirmation link in the email.

http://guides.rubyonrails.org/action_mailer_basics.html#previewing-emails

Comment (bronzdoc) [reply CONTRIBUTING.md]: If i requets that url witout a token I'm getting:

```
ActionView::Template::Error (undefined method `html_safe' for nil:NilClass
Did you mean?  html_safe?):
    3:   <%= t('.welcome_message') %>
    4:   <br />
    5:   <%= link_to t('.confirmation_link'),
    6:     update_email_confirmations_url(@user, token: @user.confirmation_token.html_safe) %>
    7: </p>
```

Comment (bronzdoc) [reply CONTRIBUTING.md]: oh i just noticed that error only happens if there's no user pending for signup

**Comment by sonalkr132** [comment]
I think this mailer preview thingy is simple enough. We can work on giving a more meaningful response instead of blowing up when no user was found. Probably it needs to be fixed [here](https://github.com/rubygems/rubygems.org/blob/master/lib/mailer_preview.rb#L3).

**Pull request merged**